backend/src/main/kotlin/app/ehrenamtskarte/backend/verification/service/CardVerifier.kt

```diff
@@ -0,0 +1,35 @@
+package app.ehrenamtskarte.backend.verification.service
+
+import app.ehrenamtskarte.backend.verification.database.repos.CardRepository
+import app.ehrenamtskarte.backend.verification.domain.Card
+import com.eatthepath.otp.TimeBasedOneTimePasswordGenerator
+import org.jetbrains.exposed.sql.transactions.transaction
+import java.time.Duration
+import java.time.Instant
+import javax.crypto.spec.SecretKeySpec
+
+val TIME_STEP: Duration = Duration.ofMinutes(5)
+const val TOTP_LENGTH = 10
+
+object CardVerifier {
+    fun verifyCardHash(cardHash: String, totp: Int): Boolean {
```

Comment:
maybe make that private?

```suggestion
    private fun verifyCardHash(cardHash: String, totp: Int): Boolean {
```

Reply:
True, done